modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc
+
+[id="cco-ccoctl-creating-at-once_{context}"]
+= Creating AWS resources with a single command
+
+If you do not need to review the JSON files that the `ccoctl` creates before modifying AWS resources, and if the process the `ccoctl` uses to create AWS resources automatically meets the requirements of your organization, you can use the `ccoctl aws create-all` command to automate the creation of AWS resources.
+
+Otherwise, you can create the AWS resources individually.
+
+//to-do if possible: xref to modules/cco-ccoctl-creating-individually.adoc for `create the AWS resources individually`
+
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To specify a directory, use the `--output-dir` flag. This procedure uses `_<path_to_ccoctl_output_dir>_` to refer to this location.
+====
+
+.Prerequisites
+
+* Extract and prepare the `ccoctl` binary.
+
+.Procedure
+
+. Extract the list of `CredentialsRequest` objects from the {product-title} release image:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc adm release extract --credentials-requests --cloud=aws --to=./credrequests quay.io/__<path_to>__/openshift-release:__<version>__
```

Comment:
it would be better to keep the content consistent with line 83 in the doc cco-ccoctl-creating-individually.adoc

`oc adm release extract --credentials-requests --cloud=aws --to=<path_to_directory_with_list_of_credentials_requests>/credrequests quay.io/<path_to>/openshift-release:<version>`